docs/build/changelog.rst

```diff
@@ -3,7 +3,7 @@ Changelog
 =========
 
 .. changelog::
-    :version: 1.3.1
```

Comment:
mmm revert this, we are still on 1.3.0.  1.3.1 is the next version.  we have automated scripts that keep this correct